lib/aims/realtime_util.py

```diff
@@ -89,6 +91,51 @@ def _pickle_filename(level_qc):
 
     return pickle_qc_file
 
+def delete_channel_id_from_pickle(level_qc, channel_id):
```

Comment:
I know you love writing code @lbesnard , but do you really need to create two new functions just to update a config file? How often do you think you'll need to do this?
More to the point, if you stored these config files in some simple text format, you could just edit them directly, without having to write/review/merge/deploy code to do it.